prototype1/prototype_v1_JC_example.yml

```diff
@@ -0,0 +1,29 @@
+author:
+  -
+    string-name:
+    surname: Colomb
+    given-names: Julien
+    email: julien.colomb@fu-berlin.de
+    PID:
```

Comment:
I would keep everything in lowercase, to make it case-insensitive and more robust towards CAPS LOCK and other capitalization errors (or is there a particular use-case where differing in the cases would be important?)

Reply:
Interesting thought, I wonder if capitalisation of name could be different in different part of the world. It would feel strance to write my name without capital letter, though.

Reply:
Note, if we ever expand to have title information, the capitalisation would be important, as some terminology requires captitalisation (for insance, foxP is a gene FoxP is a protein)

Reply:
moved to issue #22